pkg/usagemetrics/googleanalytics.go

```diff
@@ -0,0 +1,79 @@
+package usagemetrics
+
+import (
+	"github.com/golang/glog"
+	analytics "github.com/jpillora/go-ogle-analytics"
+	k8sapi "github.com/openebs/maya/pkg/client/k8s/v1alpha1"
+	openebsversion "github.com/openebs/maya/pkg/version"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8sversion "k8s.io/client-go/pkg/version"
+)
+
+const (
+	// GAclientID is the unique code of the project in Google Analytics
+	GAclientID = "UA-127388617-1"
+)
+
+// Event is a custom metric collected by OpenEBS developers
+type Event struct {
+	category string
+	action   string
+	label    string
+	// LabelSet bool
+	value int64
+	// valueSet bool
+}
+
+// NewMetric returns an Event struct with eventCategory, eventAction,
+// eventLabel, eventValue fields
+func NewMetric(c, a, l string, v int64) *Event {
+	return &Event{
+		category: c,
+		action:   a,
+		label:    l,
+		value:    v,
+	}
+}
+
+// PushEvent sends a single event to Google Analytics
+func (e *Event) PushEvent() error {
```

Comment:
Can we rename the func to `Push`?
How about `Send` as this is the naming convention in google analytics lib.

Reply:
What does Google Analytics expect? An `Event`, a `metric`, or `usage`? Based on these conventions, we might want to name our structs & functions.

Reply:
Google Analytics expects an `event`.